index.html

```diff
@@ -3598,13 +3587,13 @@ <h3>Evidence</h3>
 
         <p class="note">
 The `evidence` [=property=] provides different and complementary
-information to the `proof` [=property=]. The `evidence`
+information to the securing mechanism utlized. The `evidence`
```

Comment:
```suggestion
information to the securing mechanism utilized. The `evidence`
```

Reply:
This is the second (and last) part of one suggestion --
```suggestion
complementary to that provided by the securing mechanism in use. The `evidence`
```